functions/src/platform/assistant/handler.js

```diff
@@ -173,5 +170,5 @@ module.exports = (actionsMap) => {
     pipeline.stage(pipeline.IDLE);
   });
 
-  return functions.https.onRequest(bst.Logless.capture(functions.config().bespoken.key, app));
+  return functions.https.onRequest(bst.Logless.capture(functions.config().bespoken.key, app), dashbot.configHandler(app));
```

Comment:
according to docs you should run `dashbot.configHandler(app);` in callback but you have passed it as parameter, Do you think it should works?